src/types/index.tsx

```diff
@@ -0,0 +1,19 @@
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null;
+}
+
+export enum Error {
```

Comment:
Consider renaming `Error` enum to something more descriptive, like `FormError` or `TodoError`. 🤔